zoo.js

```diff
@@ -0,0 +1,58 @@
+//Define what an Animal is
+function Animal(name, commonName, species, location, age, image) {
+  this.name = name;
+  this.commonName = commonName;
+  this.species = species;
+  this.location = location;
+  this.age = age;
+  this.image = image;
+}
+
+//Create the animals... So many animals...
+var animals = [];
+animals.push( new Animal("Mooney", "Blue monkey", "Cercopithecus mitis", "Smith Jungle", 2, "Blue_monkey.jpg"),
```

Comment:
I don't mind the `Animal` function constructor—it keeps you from having to re-enter the key names (like `{name: 'Mooney', commonName: Blue monkey...}`)—but having so many parameters in a function definition is not ideal. It's kind of a mixed bag.

With JavaScript, it's more common to use an [Object literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer) to define objects than to use constructor functions *unless* the object has methods.